### DISCUSSION TITLE
Look for hidden directories

### DISCUSSION THREAD

**Discussion by halpomeranz** [tclahr/uac#67] [Ideas]
Outside of user home directories, directory names starting with "." are uncommon. But we'll often see attackers staging tools in directories like "/tmp/.ICEd-unix". How about adding a  check to list hidden directories that are not in user profile directories?

`find / -path /root -prune -o -path /home/\* -prune -o -type d -name .\* -print`

**Comment by tclahr** [comment]
Ok, I will convert this into an issue. Thank you for all your suggestions!
